### PR TITLE
Debug2

### PR DESCRIPTION
**Pull request by gustavocesena**
Third project at microverse, article of New York Times.

### CODE REVIEW

**Comment by bolah2009** [comment]
PR currently been reviewed

File: index.html [hunk 1]
```diff
@@ -0,0 +1,607 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
```
Comment (bolah2009) [inline index.html]: `Awesome`

Nice job using `lang="en"` to the `html` start tag, this is good for internationalisation. You should check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics).  :+1: :star2:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+            <p>
+              Andrei Linde of Stanford, a prolific theorist who first described the
+              most popular variant of inflation, known as chaotic inflation, in 1983,
+              was about to go on vacation in the Caribbean last week when Chao-Lin Kuo,
+              a Stanford colleague and a member of Dr. Kovac’s team, knocked on his door
+              with a bottle of Champagne to tell him the news.
+            </p>
+
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/ZlfIVEy_YOA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            <p class="vid_caption">Stanford Professor Andrei Linde celebrates physics breakthrough. <span class="block">Video by StanfordUniversity</span></p>
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

All images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </main>
+
+      <div class="site_index">
+        <header class="inline header_footer">
+          <img id="nyt_image" src="img/nytimes.png" alt="Not found">
```
Comment (bolah2009) [inline index.html]: Check [this comment](https://github.com/gustavocesena/positionAndFlotingElements/pull/1#discussion_r307085997)

**Comment by mosaaleb** [comment]
Reviewing

**Pull request merged**